cantera/build_devel.bat

```diff
@@ -4,6 +4,13 @@ ECHO ************************
 
 CALL scons install
 
+RD /S /Q "%PREFIX%/share/man"
+RD /S /Q "%PREFIX%/share/cantera/samples"
+
+ROBOCOPY "%SRC_DIR%/samples/cxx" "%PREFIX%/share/cantera/samples/cxx" /S /E
+DEL /F "%PREFIX%/share/cantera/samples/cxx/SCons*"
+DEL /F "%PREFIX%/share/cantera/samples/cxx/*Make*"
```

Comment:
These seem useful, what do we need to do to be able to include them?

Reply:
Not sure. `conda-build` isn't able to patch them properly, and leaving the `*.in` will confuse people. I think it's best to point users to the C++ guide and leave them out (at least for the time being). Adding a comment.